target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-vr2600v.dts

```diff
@@ -0,0 +1,416 @@
+#include "qcom-ipq8064-v1.0.dtsi"
+#include <dt-bindings/input/input.h>
```

Comment:
add a blank after #include "qcom-ipq8064-v1.0.dtsi"

<strike>Please include <dt-bindings/gpio/gpio.h> here as well and put it in front of <dt-bindings/input/input.h> to keep alphabetical order.

Use the constants GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH afterwards in stead of 1 and 0 in the gpio parameters.</strike>

Edit: GPIO_ACTIVE_LOW/GPIO_ACTIVE_HIGH are already used. by bad.